README.md

```diff
@@ -1 +1,102 @@
-# React-capston
\ No newline at end of file
+# 📗 Table of Contents
+
+- [📗 Table of Contents](#-table-of-contents)
+- [📖 Awesome books ](#-weather-)
+  - [🛠 Built With ](#-built-with-)
+    - [Key Features ](#key-features-)
+    - [Tech Stack ](#tech-stack-)
+  - [🚀 Live Demo ](#-live-demo-)
+  - [💻 Getting Started ](#-getting-started-)
+    - [Prerequisites](#prerequisites)
+    - [Setup](#setup)
+    - [Install](#install)
+    - [Usage](#usage)
+  - [👥 Author ](#-author-)
+  - [🔭 Future Features ](#-future-features-)
+  - [🤝 Contributing ](#-contributing-)
+  - [🙏 Acknowledgments ](#-acknowledgments-)
+  - [📝 License ](#-license-)
+
+# 📖 Weather <a name="about-project"></a>
+
+**Weather App** in this project, i create a weather app using React.
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Key Features <a name="key-features"></a>
+
+- **[Search your city]**
+- **[Go to detail page]**
+- **[React Router Dom]**
+
+### Tech Stack <a name="tech-stack"></a>
+
+HTML | CSS | JS | ES6 | React
+
+- More details about the project can be seen in this [video](https://www.loom.com/share/2d54f1e084ca4ed388b09794ed6a21a7) 
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+- [live link](https://weather-tcql.onrender.com)
+
+- The design was inspired [Nelson Sakwa on Behance](https://www.behance.net/sakwadesignstudio)
+
```

Comment:
- [ ] Kindly add this line under the `Acknowledgments` section, this will make the readme to look organized and professional.